Week-23/todo-list/Model/TodoModel.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+const newTodo = new mongoose.Schema({
+  todo: {
+    type: String,
```

Comment:
Don’t we have any other fields for this collection like task or label?

Reply:
as i have done just beginner question i have kept only one field for now